### PR TITLE
- Hamburger menu implementation …

### PR DESCRIPTION
**Pull request by abanseka**
Implement Hamburger Menu on Responsive design 

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -113,21 +114,60 @@ body p {
...
 
-@media (max-width: 1100px) {
+/* Small devices (portrait tablets and large phones, 600px and up) */
+@media only screen and (max-width: 600px) {
+  .btn {
+    margin: 30px;
+    padding: 5px 15px;
+    font-size: 12px;
+  }
+  body p {
```
Comment (ahamad-ch) [inline public/style.css:124]: Always apply css to specific classnames

Comment (abanseka) [reply public/style.css]: Resolved

**Review by ahamad-ch — CHANGES_REQUESTED** [review]

File: src/Components/Header/Header.js [hunk 2]
```diff
@@ -6,40 +7,56 @@ import screenImage3 from "../../images/header/screenImage3.png";
...
+
   return (
     <section className="header component-container">
       <nav className="header-banner">
         <div id="logo">
           <img src={logo} alt="text:figma land" />
         </div>
 
-        <ul className="header-menu">
+        <ul className={`header-menu ${isMenuOpen ? "open" : ""}`}>
```
Comment (ahamad-ch) [inline src/Components/Header/Header.js:21]: I can suggest you to use the `classnames` package which will make it simple here.

File: src/Components/Header/Header.js [hunk 3]
```diff
@@ -49,7 +54,11 @@ const Header = () => {
           freelancers who want a simple way to plan their schedule.
         </p>
       </div>
-      <div className={`btn-container ${isElementVisble ? "" : "hide"}`}>
+      <div
+        className={`btn-container ${classNames("isElementVisble", {
```
Comment (ahamad-ch) [inline src/Components/Header/Header.js:58]: Is there a need for the className "isElementVisible"?

File: src/Components/Header/Header.css [hunk 4]
```diff
@@ -57,16 +57,36 @@
...
+/* 
 @media (max-width: 1100px) {
   .header-banner {
     flex-direction: column;
   }
 }
 @media (max-width: 800px) {
   .dark-fade-gradient {
-    margin-top: -55vh;
+    margin-top: -50vh;
```
[the remark this answers is not in the document]
Comment (ahamad-ch) [reply src/Components/Header/Header.css:129]: I think this is not yet addressed.

**Review by ahamad-ch — APPROVED** [review]

**Pull request merged**